test/math_test.py

```diff
@@ -1141,6 +1141,20 @@ def test_project_v2_raises_if_other_is_not_iterable(self):
         # act / assert
         self.assertRaises(TypeError, v.project, other)
 
+    def test_clamp_mag_v2_max(self):
+        v1 = Vector2(7, 2)
+        v2 = v1.clamp_magnitude(5)
```

Comment:
```suggestion
        v2 = v1.clamp_magnitude(5)
        v1.clamp_magnitude_ip(5)
        self.assertEqual(v1, v2)
```
Would be nice to include `clamp_magnitude_ip` checks in integration with these tests, and the other `clamp_magnitude` tests